### PR TITLE
New macos button and separator

### PR DESCRIPTION
**Pull request by Saadnajmi**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ yarn change`

#### Description of changes

We have 2 more new controls coming in for macOS. Let's update the documentation.

### CODE REVIEW

**Comment by Saadnajmi** [comment]
It doesn't look like the failing CI is related to my change?

File: apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonImplementation.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+### Control Name
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonImplementation.md:1]: Headers should use sentence case ("Control name") not title case ("Control Name"). This is already wrong in some places but we should try to do it correctly in new content.

File: apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+Buttons are one of the core controls that make an app feel native to the platform it's on. It’s important to respect the platform's paradigms in order to help the user feel at home on macOS and keep the experience quality high.
+
+Buttons are a standard control that users can interact with to take actions within your application. The FluentUI button for macOS adds system appropriate hover effects when your mouse rests over it.
+
+<DisplayToggle onText="Dark" offText="Light" label="Theme Switcher">
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md:5]: This should use sentence case too.
```suggestion
<DisplayToggle onText="Dark" offText="Light" label="Theme switcher">
```

File: apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+### Primary Filled
+
+<img className="off" src="https://static2.sharepointonline.com/files/fabric-cdn-prod_20200701.001/fabric-website/images/controls/macos/Button/button_primaryfilled_light.png?text=LightMode" />
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md:9]: Guessing this is really feedback for @micahgodbolt but we should be careful about using such generic class names as `on` and `off` due to conflict potential.

File: apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+Buttons are one of the core controls that make an app feel native to the platform it's on. It’s important to respect the platform's paradigms in order to help the user feel at home on macOS and keep the experience quality high.
+
+Buttons are a standard control that users can interact with to take actions within your application. The FluentUI button for macOS adds system appropriate hover effects when your mouse rests over it.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md:3]: Has this text been reviewed? Each of these paragraphs feels like an overview/intro on its own, not something that makes sense to have in succession.

Comment (Saadnajmi) [reply apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md:3]: Sorta. We have an internal copy of the markdown files in fluentui-apple, that was reviewed when I checked in the button control to begin with (https://github.com/microsoft/fluentui-apple/pull/67#discussion_r446277701)

In this case, the first paragraph is a bunch of generic text I saw for other FluentUI Buttons, and the second paragraph I typed myself after getting feedback it didn't say anything specific about what the macOS FluentUI button does.

Comment (ecraig12345) [reply apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md:3]: Ah okay. Looking at places where that text appears for other platforms, I agree it's a bit strange to use it (on those platforms as well) as the entire overview. Also, it seems more like guidance to those *implementing* the Fluent Button on a given platform, not guidance for developers *using* it--and the website is supposed to be for developers *using* our controls. (This is a repeated problem.)

For comparison, this is the latest version of the button overview for web, which has been reviewed by writers (though unfortunately is not live yet):
> Buttons give people a way to trigger an action. They're typically found in forms, dialog panels, and dialogs. Some buttons are specialized for particular tasks, such as navigation, repeated actions, or presenting menus.

This is probably a more appropriate overview (aside from the "dialog panels" bit which is rather silly--not sure how that got in), and you could reuse it for macOS with minor modifications.

Comment (Saadnajmi) [reply apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md]: Updated the wording. @ecraig12345 would you or someone else be able to look at it?

File: apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md [hunk 5]
```diff
@@ -0,0 +1,20 @@
+Buttons give people a way to trigger an action. They're typically found in windows, alerts, and dialogs. Some buttons are specialized for particular tasks, such as navigation, repeated actions, or presenting menus. The FluentUI button for macOS adds system appropriate hover effects when your mouse rests over it.
```
Comment (ecraig12345) [inline apps/fabric-website/src/pages/Controls/ButtonPage/docs/mac/ButtonOverview.md:1]:
```suggestion
Buttons give people a way to trigger an action. They're typically found in windows, alerts, and dialogs. Some buttons are specialized for particular tasks, such as navigation, repeated actions, or presenting menus. The Fluent UI button for macOS adds system-appropriate hover effects when the mouse rests over it.
```

**Comment by Saadnajmi** [comment]
Thank you @ecraig12345 ! I appreciate the help and the wording fixes! I felt I was going in blind for some of this so I'm really happy to have had some insight from you on what to do / look for in future documentation =)

**Pull request merged**